src/tailwind.css

```diff
@@ -12,3 +12,11 @@
   font-family: 'Libre Franklin', sans-serif;
   color: #1c1c1c;
 }
+
+@media screen and (max-width: 640px) {
```

Comment:
Can we map this _(640px)_ to the actually corresponding tailwind media query?
_The idea is that tailwind config will be able to tell us all the breakpoints we are really using._

Reply:
Do you mean using the same breakpoint for small screens as tailwind? If so, it's the same, tailwind also uses 640px for `sm` screens

Reply:
I think what he means is using a tailwind helper for this media query. From the Tailwind docs https://tailwindcss.com/docs/functions-and-directives#screen

```css
// same as @media (min-width: 640px) {}
@screen sm {
 
}
```